src/lib/models/taskCollectionModel.js

```diff
@@ -11,7 +11,6 @@ class TaskCollectionModel extends AbstractModel {
     }
 
     addTask(task) {
-        task.addObserver('removed', this.deleteTaskById.bind(this, task.get('id')));
```

Comment:
Leave this line to make sure TaskCollectionModel.removeTaskById() works correctly